core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -144,6 +148,15 @@
   }
 
   /* All classes below should start with .conversation-skin */
+  .conversation-skin-back-to-collection-container {
+    margin-top: 25px;
```

Comment:
Also, does anything else need to be changed for mobile besides the margin? Font-size, maybe? If so, what do you propose?